include/swift/Option/Options.td

```diff
@@ -1394,6 +1394,10 @@ def sanitize_coverage_EQ : CommaJoined<["-"], "sanitize-coverage=">,
   HelpText<"Specify the type of coverage instrumentation for Sanitizers and"
   " additional options separated by commas">;
 
+def sanitize_stable_abi_EQ : Flag<["-"], "sanitize-stable-abi">,
+    Flags<[FrontendOption, NoInteractiveOption]>,
+    HelpText<"ABI instrumentation for sanitizer runtime.">;
```

Comment:
Please improve the help text here, I don't think anyone (who doesn't already know) can understand what this option does.

Reply:
I initially copied the HelpText from the same option in clang. I have updated it to make it a bit more helpful